downstream/modules/platform/ref-controller-analytics-reports.adoc

```diff
@@ -10,16 +10,17 @@ Currently available is a view-only version of the Automation Calculator utility
 
 image:aa-automation-calculator.png[Automation calculator]
 
-[NOTE]
-====
-This option is available for technical preview and is subject to change in a future release.
-To preview the analytic reports view, set the *Enable Preview of New User Interface* toggle to *On* from the *Miscellaneous System Settings* option of the {MenuAEAdminSettings} menu.
+//I don't think this is included
+//[NOTE]
+//====
+//This option is available for technical preview and is subject to change in a future release.
+//To preview the analytic reports view, set the *Enable Preview of New User Interface* toggle to *On* from the *Miscellaneous System Settings* option of the {MenuAEAdminSettings} menu.
 
-After saving, logout and log back in to access the options under the *Analytics* section on the navigation panel.
+//After saving, logout and log back in to access the options under the *Analytics* section on the navigation panel.
 
-image:aa-options-navbar.png[Navigation panel]
-====
+//image:aa-options-navbar.png[Navigation panel]
+//====
 
-Host Metrics is another analytics report collected for host data.
-The ability to access this option from this part of the UI is currently in tech preview and is subject to change in a future release.
-For more information, see the _Host Metrics view_ in xref:controller-config[{ControllerNameStart} configuration].
+//Host Metrics is another analytics report collected for host data.
```

Comment:
I see Host Metrics in the gateway UI under Automation Analytics > Host Metrics. 

I don't know whether it's still tech preview or not.

Reply:
Updated.